h2o-r/tests/testdir_misc/runit_export_parquet_multipart.R

```diff
@@ -0,0 +1,45 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+#Export file with h2o.export_file and compare with R counterpart when re importing file to check for parity.
+
+
+test.export.file <- function(path) {
+  pros.hex <- h2o.uploadFile(locate(path))
```

Comment:
` pros.hex` not only prostate

Reply:
[bca2bf8](https://github.com/h2oai/h2o-3/pull/6190/commits/bca2bf83f7b5828b4a9950d36c6cc5827e0fe92b)